plane/source/docs/guide-tailsitter.rst

```diff
@@ -147,6 +147,16 @@ The AHRS_ORIENTATION, the accelerometer calibration and AHRS trim
 should all be done for fixed wing flight. Fixed wing flight is
 considered "normal" orientation for a tailsitter.
 
+Pre-Arm Issues
+==============
+
+Since determination of roll/yaw is difficult when the nose is pointed straight up, sometimes the AHRS subsystems will not be correlated when powering up, nose up. Slight errors in compass calibration, while resulting in a successful calibration, may worsen this effect.
```

Comment:
I would say "Euler roll/yaw are indeterminate" instead of "determination of roll/yaw is difficult"
and "will disagree" instead of "will not be correlated"

Reply:
changed